Add Vue 3 docs for `disableTwoWayBinding` property. (#13894)
ckeditor/ckeditor5: docs/installation/frameworks/vuejs-v3.md

</script>
```

### `disableTwoWayDataBinding`

> Allows disabling the two-way data binding mechanism. The default value is `false`.
> 
> The reason for the introduction of this option are performance issues in large documents. After enabling this flag, the `v-model` directive, wil no longer update the connected value whenever editor's data is changed.
> 
> This option allows the integrator to disable the default behavior and only call the {@link module:core/editor/utils/dataapimixin~DataApi#getData `editor.getData()`} method on demand, which prevents the slowdowns. You can read more in the [relevant issue](https://github.com/ckeditor/ckeditor5-vue/issues/246).
> 
> ```html
> <ckeditor :editor="editor" :disableTwoWayDataBinding="true"></ckeditor>
> ```

Reviewer: ```suggestion
Allows disabling the two-way data binding mechanism. The default value is `false`.

The reason for introducing this option is performance issues in large documents. After enabling this flag, the `v-model` directive will no longer update the connected value whenever the editor's data is changed.

This option allows the integrator to disable the default behavior and only call the {@link module:core/editor/utils/dataapimixin~DataApi#getData `editor.getData()`} method on demand, which prevents the slowdowns. You can read more in the [relevant issue](https://github.com/ckeditor/ckeditor5-vue/issues/246).

```html
<ckeditor :editor="editor" :disableTwoWayDataBinding="true"></ckeditor>
```
```
